AK/Random.h

```diff
@@ -30,6 +35,11 @@ inline void fill_with_random([[maybe_unused]] void* buffer, [[maybe_unused]] siz
 #elif defined(OSS_FUZZ)
 #elif defined(__unix__) or defined(AK_OS_MACOS)
     [[maybe_unused]] int rc = getentropy(buffer, length);
+#else
+    char* char_buffer = static_cast<char*>(buffer);
+    for (size_t i = 0; i < length; i++) {
+        char_buffer[i] = std::rand();
```

Comment:
I wonder if it's worth deferring this to an impl in the cpp that calls into https://learn.microsoft.com/en-us/windows/win32/api/bcrypt/nf-bcrypt-bcryptgenrandom with a file static BCryptAlgorithmProvider for the Windows crypto rng method. I think that's a better plan than using <random>

Reply:
This is an improvement that can be done later on, no argue. But this fix is "good enough for now" no? We have bigger fishes to catch.

Reply:
I suppose. There's a lot left to be done. I think we can move forward with this.